oldeditablebuffer.go

```diff
@@ -221,7 +221,7 @@ type Undo struct {
 	buf []rune
 }
 
-// Load inserts fd's contents into File at location q0. Load will always
+// Load inserts fd's contents into OldEditableBuffer at location q0. Load will always
 // mark the file as modified so follow this up with a call to f.Clean() to
 // indicate that the file corresponds to its disk file backing.
 // TODO(rjk): hypothesis: we can make this API cleaner: we will only
```

Comment:
Ha. What was thinking not finishing my sentence. Oops. (No action requested on your part.)